src/listeners/messageListener.ts

```diff
@@ -54,6 +58,50 @@ client.on('messageCreate', async (msg) => {
     if (matchDebug) EmbedUtils.sendResponse(msg, EmbedUtils.embedColor.OK, matchDebug.title, matchDebug.footer, matchDebug.body.join('\n'));
 })
 
+/**
+ * Used for message mention stuff
+ */
+client.on('messageCreate', async (msg) => {
+
+    const regex = /https:\/\/discord.com\/channels\/(\d{18})\/(\d{18})\/(\d{18})/gm;
+    const matches = regex.exec(msg.content);
+
+    if (!matches) {
+        return;
+    }
+
+    const channelId = matches[2];
+    const messageID = matches[3];
+
+    const channel = await client.channels.fetch(channelId);
+    if (!channel || !(channel instanceof TextChannel)) {
+        return;
+    }
+
+    const message = await channel.messages.fetch(messageID);
+    if (!message) {
+        return;
+    }
+
+    // If embed, send embed
+    if (message.embeds[0]) {
+        msg.channel.send({ embeds: [ message.embeds[0] ] });
+        return;
+    }    
+
+    // Send message content
+    const content = message.content;
```

Comment:
Do we need this variable here? Couldnt line 97 use `message.content`?

Reply:
Nope, i'll remove it really quickly

Reply:
Resolved in 39e6e90